webapp/portlet/src/main/webapp/vue-apps/profile-settings/components/drawers/UserCardSettingsDrawer.vue

```diff
@@ -64,7 +64,7 @@
             outlined
             @blur="$refs.firstField.blur();" />
         </label>
-        <label for="firstField">
+        <label for="secondField">
```

Comment:
what if firstField or secondField or thirdField is not existing on front side ?

Reply:
@rdenarie those are the ids of the three selects. they should be available in the front . I may not understand your question